int8 finger
float64 value
---
bool success
